caiman_napari/main_offline_gui.py

```diff
@@ -18,6 +18,8 @@
 from .core import CaimanDataFrameExtensions, CaimanSeriesExtensions
 import os
 from .cnmf_results import show_results
+from ._cnmf import load_output_cnmf
```

Comment:
We should organize `_cnmf` and `_mcorr` into a new dir called `algorithms` or something. It will make some things easy (will explain tomorrow)